README.md

```diff
@@ -1,18 +1,38 @@
-# min-vid
-a firefox addon which lets you minimize web videos
+# Min-Vid
+Min-Vid is a Firefox add-on which lets you minimize a web video and keep it within the browser window. The video panel stays visible even when you switch tabs, so you can keep watching while you browse.
 
-## usage
+## Usage
 
-install the xpi located at [dist/addon.xpi](dist/addon.xpi) and right
-click on a video link(only youtube currently supported). In the context
+Install the xpi located at [dist/addon.xpi](dist/addon.xpi) and right
+click on a video link (only YouTube currently supported). In the context
 menu you will see an option 'Send to mini player'.
 
 ![context menu option](send-to-player.png)
 
 You can switch tabs and continue browsing as usual
 ![playing example](playing.png)
 
-This is a proof of concept, there are bugs. contributions welcome
+## Development
+This is a proof of concept, there are bugs. contributions welcome. To get started, 
+
+1.  Clone the repo:
+
+`https://github.com/meandavejustice/min-vid.git`
+
+2.  Install packages:
+
+`npm install` 
+
+3. Before submitting a Pull Request, package the Add-on:
+
+`npm run package`
```

Comment:
We may need to tweak some indentation here. Looking at the rich diff on GitHub, it looks like the numbers reset to 1. for each step. Markdown is odd and not always intuitive.